specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/examples/GetADedicatedHostGroup.json

```diff
@@ -2,7 +2,7 @@
   "parameters": {
```

Comment:
>parameters [](start = 3, length = 10)

expand was not passed as a parameter

Reply:
@dochung4 this example is still missing the $expand parameter

Reply:
Sorry I missed this one. It should be fixed now.